assets/style/style.css

```diff
@@ -358,3 +358,108 @@ body {
   align-self: center;
   justify-self: center;
 }
+
+/* ========= contact ============= */
+
+#contact-form {
+  background-image: url(../images/illustration-contact-form.svg);
+  background-repeat: no-repeat;
+  background-position: 100% 90%;
+  margin-top: 60px;
+}
+
+.contact-text-container {
+  display: flex;
+  justify-content: center;
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+.contact-text {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+}
+
+/* contact contact-form */
+
+.cform {
+  display: flex;
+  flex-direction: column;
+}
+
+.form-container {
+  margin: 0 24px 24px 24px;
+}
+
+.form-label {
+  display: none;
+}
+
+input[type=text]:focus {
+  border: 1px solid #36b37f !important;
+}
+
+form input {
+  border: 1px solid #d0d9d4;
+  margin-bottom: 24px;
+  border-radius: 5px;
+  padding: 15px;
+}
+
+form textarea {
+  border-radius: 5px;
+  border: 1px solid #36b37f;
+  padding: 15px;
+  font-family: 'Inter', sans-serif;
+}
+
+.btn {
+  display: flex;
+  flex-direction: row;
+  align-self: center;
+  justify-content: center;
+  padding: 12px;
+  width: 129px;
+  height: 48px;
+  background: #36b37f;
+  border-radius: 4px;
+  border: 0;
+  color: white;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  margin-top: 24px;
+}
+
+.btn:hover {
+  background-color: #36b37f;
+  box-shadow: 0 8px 16px rgba(54, 179, 127, 0.24);
+}
+
+.btn:active {
+  background-color: #008552;
+}
+
+.social-footer {
+  display: flex;
+  justify-content: center;
+  margin-top: 44px;
+}
```

Comment:
- Please Implement button interactions such as hover, active (when it's pressed), disabled, and enable (default state) as required by the previous reviewer and styled as shown in the Figma template. It will make your design stick more to the Figma design as required in the project  :+1: 

## see below colors 

![206037752-4d1f6c76-f50b-4c95-b94f-a6a644a66cf7](https://user-images.githubusercontent.com/95297251/206052937-2df846f2-cea0-4d4e-9935-5813f17ac6a8.png)

Reply:
Dear @MaganaAsiati , Is there anything wrong in my .btn:hover and .btn:active please? Because i can see i implement them, but you ask me to add them. can you be more specific please?

Reply:
Hi @hansderly , I believe you do not need to implement the hover on the button as requested in the previous review on the mobile version because the hover property only works with the mouse (desktop version); please accept my apologies for the error. :wink:

Reply:
Hi @Alfredbis29 , it is okay 😊
Don't worry!
I will wait for the permission to merge it.